library/bignum_core.c

```diff
@@ -53,10 +53,6 @@ size_t mbedtls_mpi_core_bitlen(const mbedtls_mpi_uint *A, size_t A_limbs)
 {
     size_t i, j;
 
-    if (A_limbs == 0) {
```

Comment:
This part conflicts with #7462

Reply:
No problem, I'll rebase once 7462 is merged

Reply:
CI says one of the checks I removed was actually necessary, anyway...